scripts/cmake/PreFind.cmake

```diff
@@ -9,15 +9,23 @@ find_program(GIT_EXECUTABLE
 
 find_package(Git REQUIRED)
 string(REPLACE "mingw64/" "" GIT_EXECUTABLE ${GIT_EXECUTABLE}) # Windows git submodule fix
-set(GIT_TOOL_PATH ${GIT_EXECUTABLE} CACHE FILEPATH "The git command line interface" FORCE)
 ### End Find Git
 
 execute_process(COMMAND ${GIT_EXECUTABLE} status
     WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}
     RESULT_VARIABLE IS_GIT_REPO
     OUTPUT_QUIET)
 if(IS_GIT_REPO GREATER 0)
-    message(FATAL_ERROR "Source code at ${PROJECT_SOURCE_DIR} is not a git "
-        "repository! Please use git to obtain the source code! See "
-        "https://www.opengeosys.org/docs/devguide/getting-started/get-the-source-code/")
+    set(IS_GIT_REPO FALSE CACHE INTERNAL "")
+    if(DEFINED OGS_VERSION)
+        message(WARNING "Using user-provided OGS_VERSION: ${OGS_VERSION}!")
+        message(WARNING "Submodule setup is skipped!")
```

Comment:
better to move the warning into the if in SubmoduleSetup.cmake.

Reply:
This handles all possibilities regarding warning output. So it would leave this here and there is no additional output necessary in subsequent CMake files.